tests/meta-mender-beaglebone-ci/conf/machine/beaglebone-mender.conf

```diff
@@ -4,6 +4,6 @@
 
 require conf/machine/beaglebone.conf
 
-MACHINEOVERRIDES =. "beaglebone:"
+MACHINEOVERRIDES =. "beaglebone:beaglebone-yocto:"
```

Comment:
If Yocto has moved away from `beaglebone` we should probably stop referring to it altogether, so I propose removing the override. Do you agree? I'm testing whether this causes any problems in #441.

Reply:
Fixing the MACHINEOVERRIDES and MENDER_DEVICE_TYPES_COMPATIBLE seem like reasonable changes to make.  I'll take a look in #441